Lib/unittest/mock.py

```diff
@@ -1600,17 +1627,20 @@ def __enter__(self):
 
         new_attr = new
 
-        self.temp_original = original
-        self.is_local = local
-        self._exit_stack = contextlib.ExitStack()
+        self._started_context = _PatchStartedContext(
+            exit_stack=contextlib.ExitStack(),
+            is_local=is_local,
+            target=self.getter(),
```

Comment:
```suggestion
            target=target,
```

You may not necessarily want to re-call self.getter() which could cause a side-effect.

Reply:
Oh, good one! I refactored a bit and overlooked this after some copy-pasting

Reply:
Interesting that `self.getter()` is also called in `get_original` method. Should I also add optional argument in here..?

Reply:
Let's try to keep the same behaviour as before unless there is a need to change.